### PR TITLE
Fix issue 19647 - Backend Marketing Store View Specific Labels Options Label not aligned

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Description (*)

Backend Marketing Store View Specific Labels Options Label not aligned
### Fixed Issues (if relevant)

1.magento/magento2#19647: Backend Marketing Store View Specific Labels Options Label not aligned

### Manual testing scenarios (*)

1.Open Backend
2.Marketing -> Cart Price Rules -> Labels -> Store View Specific Labels
3.Check Options Alignment

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-19649.instances.magento-community.engineering).
Admin access: [https://pr-19649.instances.magento-community.engineering/admin](https://pr-19649.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
Hello @speedy008. Thank you for your collaboration. I'm afraid that adding 30px margin does not solve the issue here. The space between the website/store and store view labels is calculated dynamically (try resizing the window). So even after the fix, we have the following picture:

image

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less [hunk 1]
```diff
@@ -989,7 +989,7 @@
             padding-top: 0;
             padding-right: 30px;
             width: 20%;
-            margin-left: 0;
+            margin-left: 30px;
```
Comment (rogyar) [inline app/design/adminhtml/Magento/backend/web/css/styles-old.less]: We need another solution here. This change does not resolve the issue. Please, check my comment in the conversation thread.

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by VladimirZaets** [comment]
@speedy008, I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less (hunk 1 above)
Comment (SyntaxKay) [reply app/design/adminhtml/Magento/backend/web/css/styles-old.less]: @rogyar 
Sure i will upload with another solution

**Comment by SyntaxKay** [comment]
@VladimirZaets 
I have uploaded another solution

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-19649.instances.magento-community.engineering).
Admin access: [https://pr-19649.instances.magento-community.engineering/admin](https://pr-19649.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-4307](https://jira.corp.magento.com/browse/ENGCOM-4307) has been created to process this Pull Request

**Comment by sdzhepa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, here is your [new Magento instance](https://pr-19649.instances.magento-community.engineering).
Admin access: [https://pr-19649.instances.magento-community.engineering/admin](https://pr-19649.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by sdzhepa** [comment]
Hello @speedy008 

Thank you for contribution and collaboration!

During testing, I faced an issue.
Could you please have a look and fix it?

- **Problem:** _It is not possible to deploy static-content because of changes from this PR_

**Manual testing scenario:**
1. Clone Magento based on PR code
2. Install Magento
3. Go to Magento root folder and run
`bash> php bin/magento setup:static-content:deploy -f`



**Actual Result:** Not OK,  

```
 php bin/magento setup:static-content:deploy -f

Deploy using quick strategy
adminhtml/Magento/backend/en_US         0/2185              >--------------------------- 0% %    < 1 seadminhtml/Magento/backend/en_US         1940/2185           ========================>--- 88% %   1 sec
Compilation from source: /var/www/magento/instance_2/app/design/adminhtml/Magento/backend/web/css/styles-old.less
variable @field-label-grid__column is undefined in file /var/www/magento/instance_2/var/view_preprocessed/pub/static/adminhtml/Magento/backend/en_US/css/styles-old.less in styles-old.less on line 966, column 31
964|
965|         &.admin__field-label {
966|             #mix-grid .column(@field-label-grid__column, @field-grid__columns);
967|         }
968|     }
969|

In File.php line 151:

  The contents from the "/var/www/magento/instance_2/pub/static/adminhtml/Magento/backend/en_US/css/
  styles-old.css" file can't be read. Warning!file_get_contents(/var/www/magento/instance_2/pub/stat
  ic/adminhtml/Magento/backend/en_US/css/styles-old.css): failed to open stream: No such file or dir
  ectory

```

**Comment by VladimirZaets** [comment]
@speedy008 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**